features/migrate_client_to_minion.feature

```diff
@@ -30,28 +30,21 @@ Feature: Migrate a traditional client into a salt minion
      Then the command should fail
 
   Scenario: Check that the migrated system is now a minion
+     # we should also try to migrate a traditional client in SSH push mode because of
```

Comment:
Remove the comment and put this in a card or in an issue. 

Such comments will remain forever otherwise